exchange.lua

```diff
@@ -458,6 +458,12 @@ function ex_methods.change_balance(self, p_name, delta)
 		return false, p_name .. " does not have enough money."
 	end
 
+	if delta > 0 then
+		self:log("Deposited " .. delta .. " credits", p_name)
```

Comment:
change_balance is used for things other than just deposits and withdrawals, so this logging will be incorrect sometimes.